### ISSUE TITLE
Accessibility: add aria-expanded to ExpandableText

### ISSUE THREAD

**Issue by cliu02** [bloom-housing/bloom#3490]
[ExpandableText](https://github.com/bloom-housing/ui-components/blob/main/src/actions/ExpandableText.tsx) does not have an aria-expanded attribute to determine whether the text has been expanded or not e.g. `aria-expanded={expanded}`.

Adding this improves aria accessibility, makes ExpandableText more consistent with ExpandableContent, and allows for customization of a ExpandableText toggle to change based on the existing `has-toggle` [mixin](https://github.com/bloom-housing/ui-components/blob/main/src/global/mixins.scss) for DAHLIA work.
